src/components/Tabs.react.js

```diff
@@ -0,0 +1,258 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const greyColor = '#d6d6d6';
+const offWhiteColor = '#f9f9f9';
+const primaryColor = '#1975FA';
```

Comment:
Thoughts on making these props as well? like:
```
colors = {
    border: 
    hover: 
    primary:
}
```

Reply:
Yeah, let's do it!

Reply:
;-)